bh_modules/mdcodeinline.py

```diff
@@ -0,0 +1,22 @@
+"""
+BracketHighlighter.
+
+Copyright (c) 2013 - 2015 Isaac Muse <isaacmuse@gmail.com>
+License: MIT
+"""
+
+
+def post_match(view, name, style, first, second, center, bfr, threshold):
+    """Ensure that backticks inside the inline are not highlighted."""
+    # ensure this is only used for inline markdown
+    if name != "mdcodeinline":
```

Comment:
Checking if name is `mdcodeinline` is probably not necessary since we are only attaching this plugin to mdcodeinline.